addon/services/head-tags.js

```diff
@@ -1,11 +1,11 @@
-import Ember from 'ember';
-
-const {
-  get
-} = Ember;
+import { guidFor } from '@ember/object/internals';
+import Service, { inject as service } from '@ember/service';
```

Comment:
most changes here are updating source pkgs and running a few codemods like 

https://github.com/ember-cli/ember-modules-codemod
https://github.com/rwjblue/ember-qunit-codemod